### PR TITLE
Implement arithmetic expressions in the condition parsing

### PR DESCRIPTION
**Pull request by gzsombor**
so 'where x+3 > 1 and (y*2) = 5' should work now

### CODE REVIEW

**Comment by ms705** [comment]
Hi @gzsombor!

Thanks for this -- I'll take a look, but it looks like a great addition to me.

**Comment by ms705** [comment]
Sorry for the delay! I've now had a chance to review this, and it looks great :+1: 

Since this breaks existing consumers, I'll merge this after releasing `v0.0.4` and then roll this and some other breaking changes into a new release (probably in a few weeks).

File: src/condition.rs [hunk 1]
```diff
@@ -3,11 +3,11 @@ use std::collections::{HashSet, VecDeque};
 use std::fmt;
 use std::str;
 
+use arithmetic::{arithmetic_expression, ArithmeticExpression};
 use column::Column;
 use common::{
     binary_comparison_operator, column_identifier, literal, opt_multispace, value_list, Literal,
     Operator,
-};
```
Comment (jonhoo) [inline src/condition.rs]: Removing this seems wrong?

Comment (ms705) [reply src/condition.rs]: Typo, fixing

**Comment by gzsombor** [comment]
Thanks, and welcome !

**Pull request merged**